cinema/serializers.py

```diff
@@ -114,9 +114,9 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
```

Comment:
It's a good practice to add a docstring to your methods. This helps other developers understand the purpose of the method and how it works.